azkaban_cli/azkaban.py

```diff
@@ -715,3 +716,24 @@ def fetch_executions_of_a_flow(self, project, flow, start, length):
         self.__catch_response_error(response, FetchExecutionsOfAFlowError)
 
         return response.json()
+
+    def resume_flow_execution(self, execution_id):
+        """Resume a flow execution for the Azkaban API
+
+        :param str execution_id: Execution id to be resumed
+        :return: The response from the request made
+        :rtype: requests.Response
+        :raises ResumeFlowExecutionError: when Azkaban api returns error in response
+        """
+        self.__check_if_logged()
+
+        response = api.resume_flow_execution(
+            self.__session,
+            self.__host,
+            self.__session_id,
+            execution_id
+        )
+
+        self.__catch_response_error(response, ResumeFlowExecutionError)
```

Comment:
You can add the option `ignore_empty_responses=True` here, since an empty response is actually expected when things go right (flow is resumed). This should fix the behavior and the tests should pass.

Reply:
Done.